kolibri/plugins/coach/assets/test/selectRandomExamQuestions.spec.js

```diff
@@ -5,9 +5,9 @@ jest.mock('kolibri.lib.logging');
 const EXERCISES_IDS = ['A', 'B', 'C'];
 const EXERCISES_TITLES = ['title_x', 'title_y', 'title_z'];
 const QUESTION_IDS = [
-  ['A1', 'A2', 'A3'],
-  ['B1', 'B2', 'B3', 'B4', 'B5', 'B6', 'B7', 'B8'],
-  ['C1', 'C2', 'C3', 'C4', 'C5', 'C6', 'C7', 'C8', 'C9'],
+  ['A:A1', 'A:A2', 'A:A3'],
```

Comment:
This looks a little suspicious to me - why are we updating this to the colon separated item values, when this is specifically the QUESTION_IDS with no other changes to the tests - or are we just not making the correct assertions about this in the tests for this to matter, and this should actually be the ITEM_IDS not the QUESTION_IDS?

Reply:
I made this [commit](https://github.com/learningequality/kolibri/pull/12111/commits/5b5141ccae7f18b5e79707bf2feb88a9bf09ccdd) hoping to clear up the expectations in the test & selectQuestions function call.

I have been inclined to just make `id` the key for the composite value rather than `item` and have a follow-up issue on deck to update the places that use `item` to call it `id` and add comments

Reply:
I think there might be an argument to double down on the item instead. Can double check when I am back at my desk, but it is used fairly consistently into the logging layer as well.

Reply:
See here where it ultimately ends up in the backend https://github.com/learningequality/kolibri/blob/7368f265756eed7295267cf484b3fe9581881eaf/kolibri/core/logger/models.py#L277

I think there's probably some systemic updates to be made to be more consistent.

Reply:
Awesome thanks for this! I've created this [issue](https://github.com/learningequality/kolibri/issues/12127) to align quiz creation with the use of `item` for the unique ID in particular.